### PR TITLE
Add .gitattributes

### PR DESCRIPTION
**Pull request by 1ma**
Reference: https://blog.madewithlove.be/post/gitattributes/

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/tests export-ignore
```
Comment (lcobucci) [inline .gitattributes]: Wait, this is not really correct... It is `test` instead (unless you would like to a commit to move things around)

Comment (1ma) [reply .gitattributes]: ouch, sorry :sweat_smile:

**Comment by 1ma** [comment]
Fixed. I'd rather not move the whole directory.

**Comment by lcobucci** [comment]
@1ma 🚢 I'll be releasing it ASAP, thanks for your contribution!

Regarding the "splash my nickname on everything": I'm more than happy to have more nicknames everywhere. So if you do think that `tests` is a better folder name and are willing to send a PR to change it, please do it!

**Comment by 1ma** [comment]
Ah, it's not a humbleness thingy, just a practical matter. If my name is splashed over every single line of code in the test directory `git blame` would lose a bit of its usefulness (granted that that project is very small and has just a handful of commits).

At any rate, I don't think `tests` is any better than `test`, I just happened to adapt the file from one of my packages and that mistake slipped by.

Cheers.

**Pull request merged**